enacts/wat_bal/maproom_monit.py

```diff
@@ -372,15 +372,12 @@ def wat_bal_ts(
     return ts
 
 
-def plot_scatter(ts, name, color, dash=None, customdata=None):
-    hovertemplate = "%{y} on %{x}"
-    if customdata is not None:
-        hovertemplate = hovertemplate + " %{customdata}"
+def plot_scatter(ts, name, color, dash=None):
     return pgo.Scatter(
-        x=ts["T"].dt.strftime("%-d %b"),
+        x=ts["T"].data,
```

Comment:
How did you know this `data` attribute existed? I can't find it in the documentation. It seems to be the same as `values`, which is documented.

Reply:
it actually is in the [doc](https://docs.xarray.dev/en/stable/generated/xarray.DataArray.data.html).

Reply:
Thanks. Not sure why I failed to find it.